Add instructions for linking OAuth accounts with different emails (#1118)
clerk/clerk-docs: docs/authentication/social-connections/account-linking.mdx

/>

Clerk initiates the email verification process at the outset. Upon successful email verification, regardless of the method, additional steps may be taken to validate existing connections or passwords. This is done to ensure the user's account remains secure.

## Users with different email addresses

> If a user has a different email from the one associated with the OAuth account, they can manually associate the two. To do so they must:

Reviewer: ```suggestion
If a user has a different email from the one associated with the OAuth account, they can manually associate the two. To do so, they must:
```

> 1. Sign into your Clerk application with the account that uses their main email address.

Reviewer: ```suggestion
1. Sign into their Clerk application with the account that uses their main email address.
```

> 1. In the **Manage Profile** section of their profile page, add the different email.

Reviewer: I feel like we should reference what the "profile page" is where the **Manage profile** section can be found.

Should we write "`<UserProfile />` page" or just "`<UserProfile />`"?

```suggestion
1. In the **Manage Profile** section of the [`<UserProfile />`](/docs/components/user/user-profile) page, add the different email.
```

or

```suggestion
1. In the **Manage Profile** section of the [`<UserProfile />`](/docs/components/user/user-profile), add the different email.
```

Author: Good point and in this case I like the first option you presented
